documentation/README.md

```diff
@@ -37,3 +37,52 @@ known as draw.io).
 
 When a change has been made to a `*.puml` or `*.drawio` file, you should
 re-render the diagrams using the command `task render` and commit the result.
+
+## Using GitHub to contribute
+
+No direct commits are allowed to this project. To contribute to the project you
+must follow a standard pull request based workflow. Please fork the relevant
+repositories and keep them in sync with Core. Then open a pull request from your
+fork back to core with your changes. Once a pull request is approved by one of
+the core team members it will be merged back into core.
+
+### Add remote core repository
+
+After forking a repository you can add the parent/Core repository to you local
+clone by adding a new remote.
+
+```shell
+git remote add core https://github.com/danskernesdigitalebibliotek/<REPOS>.git
+```
+
+If your local fork don't have the current release branch you should create it
+and ensure it is in sync with Core.
+
+```shell
+git checkout main
+git pull origin
+git fetch core
+git checkout -b release/x
+git rebase core/release/x release/x
+git push --set-upstream origin release/x
```

Comment:
I think it would be valuable with comments describing what the purpose of the cmds are.

Do you have a copy of the release branch in your fork? We have talked about not having the release branch on our side to prevent potential mistakes.